### DISCUSSION TITLE
Oneseco Github Newsletter

### DISCUSSION THREAD

**Discussion by Cybersoulja** [Oneseco-Media/.github#35] [Show and tell]
![image](https://github.com/Oneseco-Media/.github/assets/99073964/2af4875c-b0d1-4c4f-a861-dc824d1a25cc)

**Onboarding for the Oneseco Guild**

- The [Oneseco Guild Site](https://guild.xyz/oneseco) Link

- [ ] Document and create an onboarding path for new users

![image](https://github.com/Oneseco-Media/.github/assets/99073964/37eb6347-8499-4b4d-9542-7cfb883262bc)

**Comment by Cybersoulja** [comment]
![image](https://github.com/Oneseco-Media/.github/assets/99073964/7ec6c353-3ac8-448d-b3ed-a7e513ce6b58)

**Oneseco Guild Newsletter**

- A one page briefing for the guild as we build

- A place for guild projects to be promoted and contributors highlighted

**Reply by TrillAstro** [reply]
Ok after sleeping on it, I love this. Oneseco Guild Newsletter can be a Github Discussion to start. 

**Oneseco Guild Newsletter**

- [ ] A one page briefing for the guild as we build
